plutus-benchmark/plutus-benchmark.cabal

```diff
@@ -52,9 +52,9 @@ library plutus-benchmark-common
   other-modules:
       Paths_plutus_benchmark
   build-depends:
-      base >=4.7 && <5
-    , plutus-core -any
-    , plutus-tx -any
+      base >=4.9 && <5
+    , plutus-core ^>=1.0
```

Comment:
I mostly just left these because the benchmark library is never going to be used by anyone else. But it doesn't hurt to pin these down.

Reply:
I did this because , sometime ago when i was running outside of nix and just plain cabal,  `cabal * plutus-benchmark` started misbehaving and couldn't be satisfied by plutus-core -any. :man_shrugging: